middleware/routes/claims.js

```diff
@@ -73,4 +73,17 @@ router.post('/:vin/files', authMiddleware, getFiles, async (req, res) => {
   }
 });
 
+// Get a single file from S3.
+router.get('/:vin/file', authMiddleware, async (req, res) => {
```

Comment:
we will need this endpoint to download the file from the application, correct?

Reply:
Yep